.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+os:
+ - linux
+ - osx
+language: d
+d:
+ - dmd-beta
+ - dmd
+ - ldc
```

Comment:
OTOH, same reasoning as with the OS, we merely need to test the version we deploy with.